app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,83 @@
+
+.sponsors {
+  /* Fallback for web browsers that doesn't support RGBa */
+  background: rgb(0, 0, 0);
+  /* RGBa with 0.6 opacity */
+  background: rgba(0, 0, 0, 0.7);
+  /* For IE 5.5 - 7*/
+  filter:progid:DXImageTransform.Microsoft.gradient(startColorstr=#99000000, endColorstr=#99000000);
+  /* For IE 8*/
+  -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#99000000, endColorstr=#99000000)";
+
+  color: white;  
+  font-size: .75em;
+  height: 118px;
+  padding-top: 5px;
+  width: 192px;
+  float: left;
+
+  @include respond-to(phone) {height: 80px; width: 96px;}
```

Comment:
I could do it through Bootstrap with more divs though. This was a little solution I found to have an opacity on a background and full opacity on things inside. I'll change it - not wed to it.